libmisc/copydir.c

```diff
@@ -771,6 +772,9 @@ static int copy_file (const char *src, const char *dst,
 	        && (errno != 0))
 #endif				/* WITH_ATTR */
 	   ) {
+		if (ofd > 0) {
+			(void) close (ofd);
```

Comment:
Not doing close(ofd) in the next stanza?  or later if futimes (ofc, mt) fails?

Reply:
I missed those returning sequences. Thanks for pointing them.